dimod/binary/binary_quadratic_model.py

```diff
@@ -2439,8 +2448,25 @@ def Binaries(labels: Union[int, Iterable[Variable]],
         dtype: Data type for the returned binary quadratic models.
 
     Yields:
-        Binary quadratic models, each with a single binary variable.
-
+        A :class:`.BinaryQuadraticModel` for each binary variable.
+
+    Examples:
+        This example generates a BQM that represents a Boolean NOT gate as a
+        penalty model, :math:`2xy - x - y`, described in
+        :std:doc:`Ocean documentation's <oceandocs:index>`
+        :ref:`NOT example <oceandocs:not>`. The output of the brute-force solver,
+        :class:`~dimod.reference.samplers.ExactSolver`, shows that best
+        solutions are for assignments of the variables where :math:`z = \overline{x}`.
+
+        >>> x, y = dimod.Binaries(["x", "y"])
+        >>> bqm = 2*x*y - x - y
+        >>> print(bqm.to_polystring())
+        -x - y + 2*x*y
+        >>> print(dimod.ExactSolver().sample(bqm).lowest())
```

Comment:
I think just the first three lines are sufficient. But OK with keeping this as well.

Reply:
Also, as a fancy example, I love `(1-x)*(1-y)` as a way to show off everything symbolic in one line.

Reply:
Yes,  lovely, `(1-x)*(1-y)` I want to keep for the intro